src/AzureStack/Admin/FabricAdmin/Fabric.Admin/generate.ps1

```diff
@@ -1 +1 @@
-Start-AutoRestCodeGeneration -ResourceProvider "azsadmin/resource-manager/fabric" -AutoRestVersion "latest" -SdkRootDirectory $PSScriptRoot
\ No newline at end of file
+Start-AutoRestCodeGeneration -ResourceProvider "azsadmin/resource-manager/fabric" -AutoRestVersion "latest" -SdkRootDirectory $PSScriptRoot -ConfigFileTag "package-2018-10-01"
```

Comment:
Is there a reason why a specific tag is being passed here? Why is the code not being generated from a default tag?

Reply:
Our API spec doesn't change the default tag to the new version, we decided to do it after release, thus, we need to specific tag for generate new SDK.